mobile-map-search-and-route/src/main/java/com/esri/arcgisruntime/sample/mobilemapsearchandroute/MobileMapViewActivity.java

```diff
@@ -0,0 +1,446 @@
+                map.setDesc(currMap.getItem().getDescription());
+            } else {
+                map.setDesc(mMobileMapPackage.getItem().getDescription());
+            }
+            //check if map has transport data
+            if (currMap.getTransportationNetworks().size() > 0) {
+                map.setTransportNetwork(true);
+            }
+            //check if map has geocoding data
+            if (mMobileMapPackage.getLocatorTask() != null) {
+                map.setGeocoding(true);
+            }
+            //set map preview thumbnail
+            final ListenableFuture<byte[]> thumbnailAsync;
+            if (currMap.getItem() != null && currMap.getItem().fetchThumbnailAsync() != null) {
+                thumbnailAsync = currMap.getItem().fetchThumbnailAsync();
+            } else {
+                thumbnailAsync = mMobileMapPackage.getItem().fetchThumbnailAsync();
+            }
+            thumbnailAsync.addDoneListener(new Runnable() {
+                @Override
+                public void run() {
+                    if (thumbnailAsync.isDone()) {
+                        try {
+                            map.setThumbnailByteStream(thumbnailAsync.get());
+                            mMapPreviews.add(map);
+                        } catch (InterruptedException | ExecutionException e) {
+                            e.printStackTrace();
+                        }
+                    }
+                }
+            });
+        }
+    }
+
+    //define a simple symbol for a stop
+    private SimpleMarkerSymbol simpleSymbolForStopGraphic() {
+        SimpleMarkerSymbol simpleMarkerSymbol = new SimpleMarkerSymbol(
+                SimpleMarkerSymbol.Style.CIRCLE, Color.RED, 12);
+        simpleMarkerSymbol.setLeaderOffsetY(5);
+        return simpleMarkerSymbol;
+    }
+
+    //adds a number to stop symbol
+    private CompositeSymbol compositeSymbolForStopGraphic(
+            SimpleMarkerSymbol simpleMarkerSymbol, Integer index) {
+        TextSymbol textSymbol = new TextSymbol(12, index.toString(), Color.BLACK,
+                TextSymbol.HorizontalAlignment.CENTER, TextSymbol.VerticalAlignment.MIDDLE);
+        List<Symbol> compositeSymbolList = new ArrayList<>();
+        compositeSymbolList.addAll(Arrays.asList(simpleMarkerSymbol, textSymbol));
+        return new CompositeSymbol(compositeSymbolList);
+    }
+
+    //returns either a simple symbol or composite symbol at a given point
+    private Graphic graphicForPoint(Point point, boolean isIndexRequired, Integer index) {
+        //make symbol composite if an index is required
+        Symbol symbol;
+        if (isIndexRequired && index != null) {
+            symbol = compositeSymbolForStopGraphic(simpleSymbolForStopGraphic(), index);
+        } else {
+            symbol = simpleSymbolForStopGraphic();
+        }
+        return new Graphic(point, symbol);
+    }
+
+    //returns the graphic for a line
+    private SimpleLineSymbol routeSymbol() {
```

Comment:
I agree it can go. It just happened to be [implemented that way in the iOS sample](https://github.com/Esri/arcgis-runtime-samples-ios/blob/master/arcgis-ios-sdk-samples/Maps/Mobile%20map%20(search%20and%20route)/MobileMapViewController.swift#L88), so I replicated it. 

I suspect it may have been organized that way for clarity, since other graphics are all defined in their own methods (simpleSymbolForStopGraphic(), compositeSymbolForStopGraphic(), graphicForPoint())

Anyways, happy to change it.